src/test/java/com/google/exposurenotification/privateanalytics/ingestion/pipeline/IngestionPipelineIT.java

```diff
@@ -352,7 +352,11 @@ public void testFirestoreReaderPartitionsQueryAndReadsCorrectNumberDocuments()
     seedDatabaseAndReturnEntryVal(numDocs);
 
     PCollection<Long> numShares =
-        testPipeline.apply(new FirestoreReader(CREATION_TIME)).apply(Count.globally());
+        testPipeline
+            .apply(new FirestorePartitionQueryCreation(CREATION_TIME))
+            .apply(FirestoreIO.v1().read().partitionQuery().build())
+            .apply(FirestoreIO.v1().read().runQuery().build())
+            .apply(Count.globally());
```

Comment:
Further down below we do some assertions on some metrics in the now-deleted FirestoreConnector:
https://github.com/google/exposure-notifications-private-analytics-ingestion/blob/a57faa7cdfafc5688744ca50fc439b79b64947d0/src/test/java/com/google/exposurenotification/privateanalytics/ingestion/pipeline/IngestionPipelineIT.java#L373

https://github.com/google/exposure-notifications-private-analytics-ingestion/blob/a57faa7cdfafc5688744ca50fc439b79b64947d0/src/main/java/com/google/exposurenotification/privateanalytics/ingestion/pipeline/FirestoreConnector.java#L101

Are there equivalents in the new FirestoreIO?

Reply:
There is not a specific counter for partition count, as the PartitionQuery handler is scoped larger than a single PartitionQueryRequest. The number of Partitions resolved would map to the number of output RunQueryRequests + 1 from the PCollection. 

Is the number of partitions important, or are we just trying to ensure that it is doing something useful?

If we're just wanting to make sure a partition query rpc was issued, there is a metric we could look at for that.

Reply:
We were mostly trying to ensure it was something sensical, being our own custom implementation. Switching to The Official Connector Implementation ™ that's probably no longer a concern and we can remove the counter assertions.

Reply:
I figured out a way that we can PAssert on the number of queries returned from partition query, so we can ensure there is more than 1 partition.

Reply:
LGTM, thanks! Resolving.